Wbooks/src/app/screens/logIn/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { Formik } from 'formik';
+import PropTypes from 'prop-types';
+import { connect, useDispatch } from 'react-redux';
+import { compose } from 'redux';
+
+import AuthActions from '../../../redux/authentication/actions';
+import withLoadingScreen from '../../components/loading';
+
+import LogInScreen from './layout';
+
+function LogInContainer({ navigation }) {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(AuthActions.startUp(navigation));
+  }, [dispatch, navigation]);
+  return (
+    <Formik
+      initialValues={{ email: '', password: '' }}
+      onSubmit={({ email, password }) => {
+        dispatch(AuthActions.logIn(email, password, navigation));
+      }}
+    >
+      {({ handleChange, handleBlur, handleSubmit, values }) => (
+        <LogInScreen
+          handleChange={handleChange}
+          handleBlur={handleBlur}
+          handleSubmit={handleSubmit}
```

Comment:
Remember that the components props to handle events must be called `onChange={handleChange}` and so on... it's not a rule but a good practice